Snowiiii/Pumpkin
# Added Initial Time Packet on Login

## Description
Fixes a bug where when a player logs in, the time is shown to be incorrect for one second.


## Testing
- Logged in
- Set time to 18000
- Logout
- Login and notice time is always correct 


## Checklist
Things need to be done before this Pull Request can be merged.

- [x] All unit tests pass: `cargo test`
- [ ] I added new unit tests, so other people don't accidentally break my code by changing other parts of the codebase. [How?](https://doc.rust-lang.org/book/ch11-01-writing-tests.html)
